pkg/cluster/validation_test.go

```diff
@@ -149,3 +150,50 @@ func TestFullValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestIsNameValid(t *testing.T) {
+	shouldPass := []string{
+		"a1234",
+		"foo",
+		"12324.example.com",
+		"e123",
+		"foo_bar",
+		"foo-bar_baz-boom",
+		"foo______bar",
+		"foo bar baz",
```

Comment:
What's the reason you want to allow spaces in cluster names? Sounds like a recipe for all sorts of problems and is better to avoid.

Reply:
The direct reason was 32 unit tests failing meaning that we used that before and  our customers may be using spaces too. I allowed them to not break too much.

Reply:
🤦‍♂️